### PR TITLE
Some structural improvements, add linting, remove `types/` dir

### PR DESCRIPTION
**Pull request by paolobueno**
ping @wtrocki

### CODE REVIEW

File: apps/cloud/src/passportSecurity.ts [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-import Store from '@raincatcher/store';
-import userRouterBuilder, { User } from '@raincatcher/user';
+import * as express from 'express';
+import * as session from 'express-session';
 import * as _ from 'lodash';
+import * as passport from 'passport';
+import * as passportLocal from 'passport-local';
 
-var LocalAPIKeyStrategy = require("passport-localapikey").Strategy;
+const LocalAPIKeyStrategy: any = require('passport-localapikey').Strategy;
```
Comment (wtrocki) [inline apps/cloud/src/passportSecurity.ts:12]: 👍

File: package.json [hunk 2]
```diff
@@ -8,8 +8,9 @@
     "url": "git+https://github.com/feedhenry-raincatcher/raincatcher-typescript.git"
   },
   "scripts": {
-    "docs":"typedoc -out ./docs ./packages/",
-    "bootstrap": "lerna bootstrap"
+    "docs": "typedoc -out ./docs ./packages/",
```
Comment (wtrocki) [inline package.json:11]: This commad stopped working for me due to some internal typescript errors. Is that working for you?

File: tslint.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (wtrocki) [inline tslint.json:1]: We will need to migrate our internal eslint profile to tslint. Let's note this as separate task.

**Review by wtrocki — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (paolobueno) [reply package.json:11]: I'll give it a try

File: tslint.json (hunk 3 above)
Comment (paolobueno) [reply tslint.json:1]: yeah, the default ones for tslint are nice, there are ones that are a bit of a PITA but are nice for standardization, like ordering imports alphabetically.

File: package.json (hunk 2 above)
Comment (witmicko) [reply package.json:11]: doesnt work for me either, type errors on User

**Pull request merged**